gems/pending/appliance_console/database_replication_primary.rb

```diff
@@ -0,0 +1,45 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationPrimary < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr master register'.freeze
+
+    def initialize
+      set_defaults
+    end
+
+    def set_defaults
+      self.cluster_name              = "replication_cluster"
+      self.node_number               = 1
+      self.primary_database          = "vmdb_production"
+      self.primary_database_username = "root"
+      self.primary_database_password = nil
+      self.primary_host              = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
+    end
+
+    def ask_questions
+      clear_screen
+      say("Establish Primary Replication Server\n")
+      ask_for_cluster_name
+      ask_for_node_number
+      ask_for_primary_database_credentials
+      confirm
+    end
+
+    def activate
+      say("Configuring Primary Replication Server...")
+      already_configured? &&
```

Comment:
I don't think this method name makes sense given how it is being used.

Why would I want to configure something if it is already configured?